.vscode/launch.json

```diff
@@ -8,7 +8,14 @@
             "name": "support_sphere",
             "cwd": "src/support_sphere",
             "request": "launch",
-            "type": "dart"
+            "type": "dart",
+            // See: ../deployment/values.dev.yaml or the cloud yaml
```

Comment:
Should we be more specific about what we mean in this comment? What should we see? What should we be defining here?

Reply:
Thanks! Idk why github is marking the comments red!